src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,17 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  color: $main-accent-color;
+  font-size: $main-font-size;
+  line-height: $main-line-height;
+}
+
+* {
```

Comment:
I want to add this property really to all elements and my mentor told us we can do it

Reply:
yep, good idea, but u need `normalize` file for it, not `reset`)
`normalize` - set the default value
`reset` - reset the value